bin/mastarm-deploy

```diff
@@ -28,86 +31,157 @@ commander
+      console.error('Configurations out of sync: Need to push')
+    } else {
+      console.error('Configurations out of sync: Diverged')
+    }
+
+    // make sure there are no changes to the configurations
+    const { stdout: status } = await execa(
+      'git',
+      [
+        '-C',
+        configRepoRoot,
+        'status',
+        '-s'
+      ]
+    )
+    if (status === '') {
+      console.log('No changes to configurations repo')
+    } else {
+      console.error('Configurations out of sync: local changes exist that are not yet committed')
+      configurationsOk = false
+    }
+
+    if (!configurationsOk) process.exit(1)
+
+    configCommit = local
+
+    // decrypt env file using sops to make sure old file is overwritten with
+    // data from encoded sops file
+    const configPath = path.resolve(commander.config)
+    console.log('decrypting env file with sops')
+    const {stdout} = await execa(
+      'sops',
+      [
+        '-d',
+        path.join(configPath, 'env.enc.yml')
+      ]
+    )
+    await writeFile(path.join(configPath, 'env.yml'), stdout)
+    // at this point, we can be certain that the local configurations repo
+    // directory matches what has been committed and pushed to the remote repo
+  }
+
+  url = pkg.repository.url.replace('.git', '')
+  tag = `<${url}/commit/${commit}|${pkg.name}@${commit.slice(0, 6)}>`
+  config = loadConfig(process.cwd(), commander.config, commander.env)
+  const get = util.makeGetFn([commander, config.settings])
+
+  if (config.env.SLACK_WEBHOOK && config.env.SLACK_WEBHOOK.length > 0) {
+    logger.logToSlack({
+      channel: config.env.SLACK_CHANNEL || '#devops',
+      webhook: config.env.SLACK_WEBHOOK
+    })
+  }
+
+  const files = util.parseEntries([...commander.args, ...(get('entries') || [])])
+  util.assertEntriesExist(files)
+  const sourceFiles = files.map(f => f[0])
+  const outfiles = [...files.map(f => f[1]), ...files.map(f => `${f[1]}.map`)]
 
-const pushToS3 = createPushToS3({
-  cloudfront,
-  s3bucket
-})
+  env = get('env') || 'development'
+  minify = get('minify')
+  const buildOpts = {
+    config,
+    env,
+    files,
+    minify
+  }
+  cloudfront = get('cloudfront')
+  s3bucket = get('s3bucket')
 
-logger
-  .log(
+  const pushToS3 = createPushToS3({
+    cloudfront,
+    s3bucket
+  })
+
+  await logger.log(
     `:construction: *deploying: ${tag} by <@${username.sync()}>*
-:vertical_traffic_light: *mastarm:* v${mastarmVersion}
-:cloud: *cloudfront:* ${cloudfront}
-:hash: *commit:* ${commit}
-:seedling: *env:* ${env}
-:compression: *minify:* ${minify}
-:package: *s3bucket:* ${s3bucket}
-:hammer_and_wrench: *building:* ${sourceFiles.join(', ')}`
+    :vertical_traffic_light: *mastarm:* v${mastarmVersion}
+    :cloud: *cloudfront:* ${cloudfront}
+    :hash: *commit:* ${commit}
+    :seedling: *env:* ${env}
+    :compression: *minify:* ${minify}
+    :package: *s3bucket:* ${s3bucket}
+    :hammer_and_wrench: *building:* ${sourceFiles.join(', ')}`
```

Comment:
What purpose does this indentation change serve?

Reply:
It looks nicer to me.